### PR TITLE
Add bond-cni

### PR DESCRIPTION
**Pull request by mmirecki**
This PR will add the bond-cni to the multus installation.

Bond cni repo: [https://github.com/openshift/bond-cni](https://github.com/openshift/bond-cni)
Upstream repo: [https://github.com/intel/bond-cni](https://github.com/intel/bond-cni)

### CODE REVIEW

File: manifests/0000_70_cluster-network-operator_03_deployment-ibm-cloud-managed.yaml [hunk 1]
```diff
@@ -47,6 +47,8 @@ spec:
           value: quay.io/openshift/origin-multus-admission-controller:latest
         - name: CNI_PLUGINS_IMAGE
           value: quay.io/openshift/origin-container-networking-plugins:latest
+        - name: BOND_CNI_PLUGIN_IMAGE
+          value: quay.io/mmirecki/origin-network-interface-bond-cni:latest
```
Comment (tssurya) [inline manifests/0000_70_cluster-network-operator_03_deployment-ibm-cloud-managed.yaml:51]: why is this not an official openshift image?

**Comment by abhat** [comment]
The changes here seem OK  to me. I am concerned though about making Bond CNI a GA feature. Is Bond CNI ready to be called GA from a use-case maturity and API stability standpoint?

**Comment by mmirecki** [comment]
> The changes here seem OK to me. I am concerned though about making Bond CNI a GA feature. Is Bond CNI ready to be called GA from a use-case maturity and API stability standpoint?

This was tested by the CNF team and no issues were found so far.
Note that we are only supporting the sriov use-case.

**Comment by abhat** [comment]
/approve

We still need to make a call about API stability and use-case maturity. Testing coverage by CNF team is orthogonal to how customers plan to use the feature and if the API itself seems stable.

**Pull request merged**